qgreenland_dowload.py

```diff
@@ -559,35 +598,31 @@ def download_data(self):
                 if layer['id'] == parent:
                     for asset in layer['assets']:
                         files_to_download.append(asset['file'])
+                        total_size+=asset['size_bytes']
+                        print(asset['size_bytes'])
                     layer_to_download[parent] = files_to_download
+        
+        # get the bytes in megabytes
+        total_size = total_size / 1000000
```

Comment:
I recently learned about a trick to make large numbers more readable in python. You can break them up in to digit groups using underscores: `1_000_000` is the same as `1000000` :)

Reply:
oh thanks! I didn't know it!